bxin/ptg2m1m3/m1m3_diagnostic.ipynb

```diff
@@ -6,8 +6,11 @@
    "source": [
```

Comment:
<p>If I got Russel right, we should use aget only the first time and the .get. Leave this comment here for all the coming aget :-)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>